partitura/io/importmei.py

```diff
@@ -728,6 +744,53 @@ def _handle_mrest(self, mrest_el, position, voice, staff, part):
         )
         # add mrest to the part
         part.add(rest, position, position + parts_per_measure)
+        # return duration to update the position in the layer
+        return position + parts_per_measure
+    
+    def _handle_multirest(self, multirest_el, position, voice, staff, part):
+        """
+        Handles a rest that spawn multiple measures
+
+        Parameters
+        ----------
+        multirest_el : lxml tree
+            A mrest element in the lxml tree.
+        position : int
+            The current position on the timeline.
+        voice : int
+            The voice of the section.
+        staff : int
+            The current staff also refers to a Part.
+        part : Partitura.Part
+            The created part to add elements to.
+
+        Returns
+        -------
+        position + duration : int
+            Next position on the timeline.
+        """
+        # find id
+        multirest_id = multirest_el.attrib[self._ns_name("id", XML_NAMESPACE)]
+        # find how many measures
+        n_measures = int(multirest_el.attrib["num"])
+        if n_measures > 1:
+            raise Exception(f"Multi-rests with more than 1 measure are not supported yet. Found one with {n_measures}.")
```

Comment:
This is function is described as handle for rests that spawn multiple measures but if the measures are more than one it raises an exemption, is that intended?

Reply:
Yes, for the way the MEI and the parsing code are structured, handling multiple measures  in an error-robust way would require some major effort, which I will do for another release